socialforce/stateutils.py

```diff
@@ -7,7 +7,9 @@ def desired_directions(state):
     """Given the current state and destination, compute desired direction."""
     destination_vectors = state[:, 4:6] - state[:, 0:2]
     norm_factors = np.linalg.norm(destination_vectors, axis=-1)
-    return destination_vectors / np.expand_dims(norm_factors, -1)
+    desired_directions = destination_vectors / np.expand_dims(norm_factors, -1)
```

Comment:
This actually fails some code checks (cannot name variable like the function). I like your name, but maybe just name it `directions`?